lib/api/CBoostedTreeInferenceModelBuilder.cc

```diff
@@ -160,11 +160,19 @@ CClassificationInferenceModelBuilder::CClassificationInferenceModelBuilder(
     TStrVec fieldNames,
     std::size_t dependentVariableColumnIndex,
     const TStrVecVec& categoryNames)
-    : CBoostedTreeInferenceModelBuilder(fieldNames, dependentVariableColumnIndex, categoryNames) {
+    : CBoostedTreeInferenceModelBuilder(fieldNames, dependentVariableColumnIndex, categoryNames),
+      m_ClassificationLabels{categoryNames[dependentVariableColumnIndex]} {
 }
 
 void CClassificationInferenceModelBuilder::setAggregateOutput(CEnsemble* ensemble) const {
     ensemble->aggregateOutput(std::make_unique<CLogisticRegression>(ensemble->size(), 1.0));
 }
+
+CInferenceModelDefinition&& CClassificationInferenceModelBuilder::build() {
+    auto&& definition{this->CBoostedTreeInferenceModelBuilder::build()};
```

Comment:
nit: I prefer using assignment with auto since this will often deduce `std::initializer_list` which is rarely wanted
```suggestion
    auto&& definition = this->CBoostedTreeInferenceModelBuilder::build();
```

Reply:
and below

Reply:
Done